src/main/java/ch/kalunight/zoe/util/FullTierUtil.java

```diff
@@ -1,92 +1,130 @@
 package ch.kalunight.zoe.util;
 
 import ch.kalunight.zoe.exception.NoValueRankException;
+import ch.kalunight.zoe.model.GameQueueConfigId;
 import ch.kalunight.zoe.model.player_data.FullTier;
 import ch.kalunight.zoe.translation.LanguageManager;
 import net.rithms.riot.api.endpoints.league.dto.LeagueEntry;
 
 public class FullTierUtil {
 
+  private static final int BO_LEAGUE_POINTS = 100;
+
   private FullTierUtil() {
     // hide default public constructor
   }
-  
-  public static String getTierRankTextDifference(LeagueEntry oldEntry, LeagueEntry newEntry, String lang) {
-    
+
+  public static String getTierRankTextDifference(LeagueEntry oldEntry, LeagueEntry newEntry, String lang, GameQueueConfigId queue) {
```

Comment:
Codacy found an issue: [The method 'getTierRankTextDifference(LeagueEntry, LeagueEntry, String, GameQueueConfigId)' has an NPath complexity of 481, current threshold is 200](https://app.codacy.com/manual/KaluNight/Zoe-Discord-Bot/pullRequest?prid=6283923)